test/unit/model/export/SingleDeliverySqlResultExporterTest.php

```diff
@@ -0,0 +1,130 @@
+use oat\generis\test\TestCase;
+use oat\taoOutcomeUi\model\export\ColumnsProvider;
+use oat\taoOutcomeUi\model\export\SingleDeliverySqlResultsExporter;
+use oat\taoOutcomeUi\model\ResultsService;
+use oat\taoOutcomeUi\model\table\GradeColumn;
+use oat\taoOutcomeUi\model\table\ResponseColumn;
+use taoResultServer_models_classes_Variable as Variable;
+
+class SingleDeliverySqlResultExporterTest extends TestCase
+{
+    private $dataFixture = [
+        [
+            'Test Taker ID' => 'http://nec-pr.docker.localhost/tao.rdf#i5f16bd028eb6e202ad4b5d184\'f67e22',
+            'Compilation Time' => 1594828375,
+            'Field Without Type' => 12345
+        ],
+        [
+            'Test Taker ID' => 'http://nec-pr.docker.localhost/tao.rdf#i5f16bd028eb6e202ad4b5d43f67e24',
+            'Compilation Time' => 1594828388,
+            'Field Without Type' => 33333
+        ],
+        [
+            'Tets Variable Grade Column' => 'http://nec-pr.docker.localhost/tao.rdf#i5f16bd028eb6ee202ae213123121231',
+            'Compilation Time' => 159482838228,
+            'Field Without Type' => 3311333
+        ],
+        [
+            'Tets Variable Response Column' => 'http://nec-pr.docker.localhost/tao.rdf#i5f16bd028eb6ee202ae11111',
+            'Compilation Time' => 1594838228,
+            'Field Without Type' => 331111
+        ]
+    ];
+
+    /**
+     * @throws \common_exception_NotFound
+     */
+    public function testGetExporterAndGetExportData()
+    {
+        $deliveryMock = $this->createMock(\core_kernel_classes_Resource::class);
+        $deliveryMock->method('exists')->willReturn(true);
+
+        $resultServiceMock = $this->createMock(ResultsService::class);
+
+        $columnsProviderMock = $this->createMock(ColumnsProvider::class);
+
+        $modelMock = $this->createMock(Ontology::class);
+        $modelMock->expects($this->once())
+            ->method('getResource')
+            ->willReturn($deliveryMock);
+
+        $singleDeliveryExporter = new SingleDeliverySqlResultExporterMock($modelMock, $deliveryMock, $resultServiceMock, $columnsProviderMock);
+
+        $variableColumnsToExport = [
+            new GradeColumn(
+                '',
+                'Tets Variable Grade Column',
+                '',
+                Variable::TYPE_VARIABLE_IDENTIFIER),
+            new ResponseColumn(
+                '',
+                'Tets Variable Response Column',
+                '',
+                'nonexistent type')
+        ];
+        $singleDeliveryExporter->setFixtureColumnsToExport($variableColumnsToExport);
+
+        $exporter = $singleDeliveryExporter->getExporterMock($this->dataFixture);
+
+        $sqlExpected = file_get_contents(__DIR__ . 'sqlFixture.sql');
+
+        $sql = $singleDeliveryExporter->getExportDataMock($exporter);
+
+        $this->assertEquals($sqlExpected, $sql);
+    }
+
+}
+
+class SingleDeliverySqlResultExporterMock extends SingleDeliverySqlResultsExporter
```

Comment:
No need for `implements ResultsExporterInterface`

it's already implemented in SingleDeliveryResultsExporter